docs/excel/excel-add-ins-error-handling.md

```diff
@@ -63,8 +66,12 @@ The following table is a list of errors that the API may return.
 |`InvalidSelection`|The current selection is invalid for this operation.|
 |`ItemAlreadyExists`|The resource being created already exists.|
 |`ItemNotFound` |The requested resource doesn't exist.|
-|`NotImplemented`  |The requested feature isn't implemented.|
+|`nonBlankCellOffSheet`|Can't insert new cells because it would push non-empty cells off the end of the worksheet. These cells might appear empty but have blank values, some formatting, or a formula. Delete enough rows or columns to make room for what you want to insert and then try again.|
```

Comment:
Should this `error.code` value be capitalized?
```suggestion
|`NonBlankCellOffSheet`|Can't insert new cells because it would push non-empty cells off the end of the worksheet. These cells might appear empty but have blank values, some formatting, or a formula. Delete enough rows or columns to make room for what you want to insert and then try again.|
```